scripts/utilities/retry.js

```diff
@@ -0,0 +1,19 @@
+function Retry(
```

Comment:
If this is used in only one script and doesn't have tests, is it worth having this utility in its own file?

Reply:
I really don't mind where it lives, I did this as it might be useful in the future and doesn't hide the function in a file that is unrelated to it.